_components/customisation.md

```diff
@@ -0,0 +1,199 @@
+
+Taking `o-table` as an example, the "master" brand version offers a "row stripes" feature, which uses "master" brand colours such as "paper" and "wheat":
+{% include demo.html component="o-table" demo="row-stripes" brand="master" height="242px" %}
+
+The "internal" brand version also supports "row stripes" but uses a different colour palette:
+{% include demo.html component="o-table" demo="row-stripes" brand="internal" height="242px" %}
+
+The "whitelabel" brand does not support "row stripes" and therefore outputs no styles to support that feature:
+{% include demo.html component="o-table" demo="basic" brand="whitelabel" height="256px" %}
+
+### Choose A Brand
+
+The "master" brand is the default brand, but your project may use any of the provided Origami brands. The brand is configured once and affects all components used by your project.
+
+To select a brand within your project, see the tutorial for your method of including Origami components:
+
+- [Build service tutorial](/docs/tutorials/build-service/#selecting-a-brand)
+- [Manual build tutorial](/docs/tutorials/manual-build/#selecting-a-brand)
+
+### Customise A Brand
+
+If your project has access to a component's Sass API (see the [manual build tutorial](/docs/tutorials/manual-build/#selecting-a-brand)) it is possible to customise your chosen brand. For example, at the time of writing, the [`oTypographySetFont`](https://registry.origami.ft.com/components/o-typography@6.4.5/sassdoc?brand=master#mixin-otypographysetfont) mixin can be used to customise component fonts. This customises the font used by all components including in your project.
+
+<aside>
+Not all components support brand customisation but the Origami team are happy to add support as needed. Ask the core Origami team for new features in the <a href="https://financialtimes.slack.com/messages/{{site.data.contact.slack}}">#{{site.data.contact.slack}}</a> Slack channel.
+</aside>
+
+<pre><code class="o-syntax-highlight--scss">// Incomplete example for demo purposes only
+
+// Set the brand.
+$o-brand: 'whitelabel';
+
+// Customise: Tell the o-typography component to use our custom
+// font, it will affect all other components which
+// as they are built using o-typography.
+@include oTypographySetFont(
+    $type: 'sans',
+    $family: 'Comic Sans MS', sans-serif
+);
+
+// Customise: Tell the o-header-services we want hot pink
+// in the navigation.
+@include oHeaderServicesCustomize((
+	'nav-hover-background': hotpink
+));
+</code></pre>
+
+
+<figure>
+	<img alt="" src="/assets/images/component-customisation/whitelabel-customisation.png" />
+	<figcaption>
+        An example project which builds on the whitelabel brand. The font has been customised with `o-typography` Sass (oooh! Comic Sans). It affects all components which use `o-typography`, including the other components on the page, `o-table` and `o-header-services`. `o-header-services` has also been customised using Sass to have a unique highlight colour (oooh! hotpink!)
```

Comment:
Good caption 😄 

I checked how this renders in HTML and the backticks are present. Did you want the backticks converted to `<pre><code>` here?

![image](https://user-images.githubusercontent.com/1569131/111619948-0c131a00-87de-11eb-859b-759dbc77212e.png)

Reply:
So I noticed and didn't wrap in `code` out of laziness 🙈 But I actually think we should leave it like this. Visually it's not great and `code` isn't doing much semantically in this context around component names.
![Screenshot 2021-03-19 at 16 27 23](https://user-images.githubusercontent.com/10405691/111812505-19a8cc80-88d0-11eb-8422-81cbdd0bf7b5.png)